mdx_oembed/endpoints.py

```diff
@@ -6,6 +6,14 @@
     'https?://youtu.be/*',
 ])
 
+SLIDESHARE = oembed.OEmbedEndpoint('http://www.slideshare.net/api/oembed/2', [
+    'http://www.slideshare.net/*/*',
```

Comment:
@rennat when i said lets source from there that means i was talking about us sourcing once and putting it in the code not the enduser making call everytime. Hope that makes it clear. So back to my question, what else do you need from my side to put this in prod/release. I am kind of hoping to get this modification made available on pip so that some of my other projects can directly make use of this without modifications to core.